viewing_party/party.py

```diff
@@ -1,23 +1,122 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if not title or not genre or not rating:
+        return None
+    else:

```

Comment:
No need to use an else here since if the guard clause returns, we definitely want to execute the code here. We can then have less indentation, which is easier to read!